releasenotes/notes/measurement_property_bug_fix-12461088823a943c.yaml

```
@@ -1,19 +1,11 @@
 ---
-prelude: >
-    This release fixes a bug in the initialisation of a DynamicsBackend when a Target object was provided for the initialisation.
-    Furthermore, it also features the possibility to automatically load a control_channel_map from an existing backend when using 
-    DynamicsBackend.from_backend() method. 
 features:
   - |
-    Initialising a DynamicsBackend instance using the from_backend() method now loads automatically the
-    control_channel_map of the original backend to the DynamicsBackend option "control_channel_map".
-    Note that if the user sets manually the option "control_channel_map", the provided option will overwrite 
-    The default map from the original backend. 
+    The :meth:`DynamicsBackend.from_backend` method has been updated to automatically populate the 
+    `control_channel_map` option based on the supplied backend if the user does not supply one.
 
 fixes:
   - |
-    A bug regarding the setting of the measurement properties at the initialisation of the
-    DynamicsBackend instance was always resetting the measurement instruction to a default setting 
-    even though there was already a measurement instruction stored in the provided Target object. This bug is now fixed
-    and the measurement instruction is added only if the designated qubit in the provided Target does not have a measurement instruction.
+    A bug in :meth:`DynamicsBackend.__init__` causing existing measurement instructions for a user-supplied
+    :class:`Target` has been fixed.
```

Apologies this sentence is incomplete :).

```suggestion
    A bug in :meth:`DynamicsBackend.__init__` causing existing measurement instructions for a user-supplied
    :class:`Target` to be overwritten has been fixed.
```